orbit-2-rxjava2/README.md

```diff
@@ -0,0 +1,87 @@
+# Orbit 2 RxJava2 plugin
+
+The RxJava 2 plugin provides:
+
+- RxJava 2 DSL operators
+
+## Including the module
+
+```kotlin
+implementation("com.babylon.orbit2:orbit-rxjava2:<latest-version>")
+```
+
+## RxJava 2 DSL Operators
+
+The Core DSL contains the following operators:
+
+- transformObservable
+- transformSingle
+- transformMaybe
+- transformCompletable
+
+### transformObservable
+
+``` kotlin
+class ExampleViewModel : ContainerHost<ExampleState, ExampleSideEffect> {
+    ...
+
+    fun startLocationTracking() = orbit {
+            transformObservable { subscribeToLocationUpdates() }
+                .reduce { state.copy(currentLocation = event) }
+        }
+    }
+}
+```
+
+You can use this operator to subscribe to hot or cold observables.
+This operator acts similar to `flatMap`.
```

Comment:
as we are talking about RxJava, is it worth linking to the RxJava documentation for `flatMap` for those that aren't sure what it is?